frontend/src/components/problem/ProblemDisplay.tsx

```diff
@@ -40,11 +35,31 @@ import {
 import Loading from '../core/Loading';
 import ErrorMessage from '../core/Error';
 import { FlexBareContainer } from '../core/Container';
-import { generateRandomId, validIdentifier } from '../../util/Utility';
+import { generateRandomId } from '../../util/Utility';
 import { HoverTooltip } from '../core/HoverTooltip';
 import { Coordinate } from '../special/FloatingCircle';
 import ProblemTags from './ProblemTags';
 import { useAppSelector, useProblemEditable } from '../../util/Hook';
+import ProblemInputDisplay from './ProblemInputDisplay';
+import { ProblemHelpModal } from '../core/HelpModal';
+
+const defaultDescription: string = [
+  'Replace this line with a short description.',
+  '### Input',
+  'This problem accepts X parameter(s): `name` (Integer).',
+  '1. `name`: Replace this with a description of the parameter.',
+  '### Output',
+  'This problem outputs a(n) Integer.',
+  '\\',
+  'Replace this with an explanation of the output, if necessary.',
+  '### Example Test Case',
+  '**Input**',
+  '`name`: `5`',
+  '**Output**',
+  '`1`',
+  '**Explanation**',
+  'Replace this line with an explanation of this example test case.',
+].join('\n\n');
```

Comment:
Another thing I was thinking as well was thoughts on simplifying this template a bit? Looking at LeetCode and Binary Search (and to a lesser extent Practice It), generally they include the parameters in the description (given X and Y parameters, write a method Z that returns W) and then simply have a section for example test cases (as we do now), which I think might be easier for teachers to understand if they end up writing simpler problems

Reply:
I agree - I'll play around with this and remove some of the unnecessary lines.

Reply:
What do you think about the following change? I removed everything mentioned except the "Parameters" list - I could remove that too, I think that can be helpful though, let me know.

**New Template Description**
<img width="1280" alt="New Template Description" src="https://user-images.githubusercontent.com/7987279/120548730-8730a900-c3a7-11eb-92b9-e49d8f2fcc73.png">

Reply:
Sure, I think this looks good - I guess depending on teacher feedback if people have any confusions we can continue to refine this once we get to that step

Reply:
Ok cool, sounds great! 😃